optmod/coptmod/evaluator.c

```diff
@@ -8,6 +8,7 @@
  */
 
 #include "evaluator.h"
+#include <inttypes.h>
```

Comment:
Since these stdlib headers are called in the header file which is already being "included" it isn't necessary to include them in the c files. I'll take care of this just wanted to let you know.